src/components/notifications/notificationMenu.tsx

```
@@ -0,0 +1,267 @@
+                      type="button"
+                      onClick={() => setShowFilter((prev) => !prev)}
+                      className="flex items-center gap-1.5 rounded-[10px] py-2 text-[14px] leading-[14px] text-[#828282]"
+                    >
+                      {filter === "all"
+                        ? t("notifications.filters.all")
+                        : t("notifications.filters.unread")}
+                      <svg
+                        className="h-4 w-4"
+                        viewBox="0 0 20 20"
+                        fill="currentColor"
+                      >
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)